assets/src/scss/blocks/ActionsList.scss

```diff
@@ -1,30 +1,32 @@
 .actions-list {
-  margin-bottom: $sp-4;
+  // TODO: Review this logic and standarize the code between all layouts
```

Comment:
We'll need to follow up this part whenever we decide to integrate the carousel not only to actions but also to other posts.